plugins/cherwell/Dockerfile

```diff
@@ -1,25 +1,20 @@
-FROM komand/python-3-plugin:2
-# The three supported python parent images are:
-# - komand/python-2-plugin
-# - komand/python-3-plugin
-# - komand/python-pypy3-plugin
-#
-LABEL organization=komand
+FROM --platform=linux/amd64 rapid7/insightconnect-python-3-slim-plugin:5
```

Comment:
I think we're trying to adopt specifying the full version in this file to keep what we test/release on the same version?